src/Data/Entities/AdditionalFacilityEntity.cs

```diff
@@ -1,13 +1,13 @@
 ﻿using System.Collections.Generic;
 
-namespace Data.Models
+namespace Data.Entities
 {
-    public class AdditionalServiceEntity : Entity<int>
+    public class AdditionalFacilityEntity : Entity<int>
     {
         public string Title { get; set; }
         public decimal Price { get; set; }
         public string Description { get; set; }
 
-        public virtual ICollection<OrderEntity> Orders { get; set; }
+        public virtual ICollection<BookingEntity> Bookings { get; set; }
```

Comment:
We don't need virtual properties(virtual word) in ef entities, it doesn't work by default - read about it
It is the same in all other cases.